examples/closure-app/silabs/src/AppTask.cpp

```diff
@@ -72,7 +74,7 @@ namespace app {
 namespace Clusters {
 namespace ClosureControl {
 
-static chip::BitMask<Feature> sFeatureMap(Feature::kCalibration);
+static chip::BitMask<Feature> sFeatureMap(Feature::kCalibration, Feature::kPositioning, Feature::kMotionLatching, Feature::kSpeed);
```

Comment:
I lost track of this during the review and changes. Why is this a generic static variable in AppTask within the closureControl Namespace. Shouldn't the featuremap be set in the cluster server class?